ui/src/components/Tasks/DailyTasks/index.tsx

```diff
@@ -64,13 +64,14 @@ const selectMoreOptions = [
 ];
 
 // const NoTodaysCommitment: React.FC<{ addItemCb: () => void }> = (p) => (
-const NoTodaysCommitment = () => (
+const NoTodaysCommitment: React.FC<{ openSlidebar: () => void }> = (p) => (
```

Comment:
Callback props should be named with `on` prefix. This should be called `onOpenSlidebar`. But a better name here can probably be `onAddCommitment`. 
It's better if we communicate the intention of the action instead of what the UI is going to look like. There is a good chance that we'll change the UI at some point, e.g it might become something which is not opening a slidebar, but a popup instead.